draft-ietf-quic-applicability.md

```diff
@@ -230,14 +233,14 @@ translation could still affect the path. In particular, firewalls will often
 not admit server traffic for which it has not kept state for corresponding
 packets from the client.
 
-A QUIC application can adjust idle periods to manage the risk of timeout
-(noting that idle periods and the network idle timeout is distinct from the
-connection idle timeout, defined as the minimum of the idle timeout parameter
-in Section 10.1 of {{QUIC}}), but then there are three options:
+A QUIC application can adjust idle periods to manage the risk of timeout (noting
+that idle periods and the network idle timeout is distinct from the connection
+idle timeout, defined as the minimum of either endpoint's idle timeout
+parameter; see {{Section 10.1 of QUIC}}), but then there are three options:
```

Comment:
This sentence is long, can we break it up?

```suggestion
A QUIC applications can adjust idle periods to manage the risk of timeout.
Idle periods and the network idle timeout are distinct from the connection
idle timeout, which is defined as the minimum of either endpoint's idle timeout
parameter; see {{Section 10.1 of QUIC}}). There are three options:
```

Reply:
"A QUIC applications" => "QUIC applications" or "A QUIC application"